autoload/SpaceVim/layers/lang/c.vim

```diff
@@ -88,6 +88,10 @@ function! SpaceVim#layers#lang#c#plugins() abort
 endfunction
 
 function! SpaceVim#layers#lang#c#config() abort
+  call SpaceVim#mapping#gd#add('c',
+        \ function('s:go_to_def'))
```

Comment:
we should defind two function, here is a example, user who enable lsp for c but not for cpp. he should be able to use lsp gd